src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_TicketingStack/FareContracts/Component/TransportMode.tsx

```diff
@@ -29,24 +60,42 @@ const TransportMode = ({
           disabled={disabled}
         />
       ))}
-      <ThemeText type="label__uppercase" color={'secondary'}>
-        {modes
-          .map((tm) => t(FareContractTexts.transportMode(tm.mode)))
-          .join('/')}
+      {modesCount > modesDisplayLimit && (
+        <View style={[styles.multipleModes, boxHeight]}>
+          <ThemeText color={'transport_other'} type="label__uppercase">
+            +{modesCount - modesDisplayLimit}
+          </ThemeText>
```

Comment:
The accessibility is only telling +2. Should it say something like: `Several modes`?